packages/design-system-tokens/src/themes/medicare-component.ts

```diff
@@ -412,18 +421,21 @@ export const components: AnyTokenValues = {
     '__line-height':                              '1.3',
     '__background-color--disabled':               t.color['gray-lighter'],
     '__border-width':                             '2px',
+    '__border-width--disabled':                   '1px',
     '__border-width--error':                      '3px',
     '__border-width--success':                    '3px',
     '__border-color':                             t.color['gray'],
     '__border-color--disabled':                   t.color['gray-darker'],
     '__border-color--error':                      t.color['error'],
     '__border-color--error--inverse':             t.color['error-lighter'],
-    '__border-color--inverse':                    t.color['black'],
+    '__border-color--inverse':                    t.color['gray'],
```

Comment:
I didn't mean to change this, but I kept getting visual regressions on datepicker inverse without setting this. Since prod doesn't show a black border for mgov text-input I figured this was a fix - thoughts?

Reply:
This sounds like a good fix. I looked at the inverse dropdown on main and this branch, and they're the same gray color. Really I don't think the gray gives very good contrast on a dark background, but I don't think it should be changed right now.